README.md

```diff
@@ -34,11 +34,21 @@ Feel free to try out and let us know if you cannot wait.
 
 ## Install dockert3kit into your distribution
 
-Add `lauri/dockert3kit` as dev dependency in your composer, using the latest stable release is highly recommended.
+Add `lauri/dockert3kit` as dev dependency in your composer.
+
+***Note***
+
+**If you are using EXT:solr = 6.0.0 please use the 6.0.0 release**
+
+*Example*:
+
+    composer require --dev lauri/dockert3kit '~6.0.0'
+
+**If you are using EXT:solr = 6.1.0 please use the 6.1.0 release**
 
 *Example*:
 
-    composer require --dev lauri/dockert3kit '~3.0.0'
+    composer require --dev lauri/dockert3kit '~6.1.0'
```

Comment:
@laurisaarni ok, but the change I made for ext-solr 6.0.0 branch. If we call tag the 6.0.0 branch as 4.0.0 do we still keep the 3.0.0 tag for the master branch (the solr v6.1.0 support) i.e 
solr 6.0.0 => 4.0.0 release
solr 6.1.0 => 3.0.0 release